fedmsg/consumers/ircbot.py

```diff
@@ -208,12 +210,13 @@ def __init__(self, hub):
             pretty = settings.get('make_pretty', False)
             terse = settings.get('make_terse', False)
             short = settings.get('make_short', False)
+            rate = settings.get('line_rate', 1.0)
```

Comment:
It is.  I just wanted to prove to myself that I could make them different and it could work.  ;p